### PR TITLE
Proj0215: Provide a compliant NuGet package icon

### PR DESCRIPTION
**Pull request by Corniel**
Documentation for https://github.com/dotnet-project-file-analyzers/dotnet-project-file-analyzers/pull/180

### CODE REVIEW

File: general/nuget-packages.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ packages.
 | [Proj0213](../rules/Proj0213.md) | Consider adding the icon url for maximum compatibility.
 | [Proj0214](../rules/Proj0214.md) | Do include the package id.
 | *none*                           | Consider choosing a NuGet package name with a prefix that meets NuGet's [prefix reservation criteria](https://learn.microsoft.com/nuget/nuget-org/id-prefix-reservation).
-|                                  | Do use an image that is 128x128 and has a transparent background (PNG) for best viewing results.
+| [Proj0215](../rules/Proj0215.md) | Do use an image that is 128x128 and has a transparent background (PNG) for best viewing results.
```
Comment (LauraXiulan) [inline general/nuget-packages.md:26]: Shouldn't this be above *none* as it is the next rule after 0214?

File: rules/Proj0215.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+# Proj0215: Provide a compliant NuGet package icon
+To ensure the creation of well-formed [NuGet packages](../general/nuget-packages.md),
+use an image that is 128x128 and has a transparent background (PNG) for best
```
Comment (LauraXiulan) [inline rules/Proj0215.md:3]:
```suggestion
use an image that is 128x128 and has a transparent background (PNG) for the best
```

File: general/nuget-packages.md (hunk 1 above)
Comment (Corniel) [reply general/nuget-packages.md:26]: No, I'd like to keep it in order with how Microsoft stated it.

Comment (LauraXiulan) [inline general/nuget-packages.md:26]:
```suggestion
| [Proj0215](../rules/Proj0215.md) | Do use an image that is 128x128 and has a transparent background (PNG) for the best viewing results.
```

**Review by LauraXiulan — APPROVED** [review]

File: general/nuget-packages.md (hunk 1 above)
Comment (CptWesley) [inline general/nuget-packages.md:26]: Max 1MB is also part of the spec

Comment (Corniel) [reply general/nuget-packages.md]: Addressed.

**Pull request merged**